# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender_build/src/shader_features.rs
# Commit: c0f58fd74e56
# Full Hash: c0f58fd74e56b24f67fccae08097d151e21a0f80
# Author: Ashley Hale <ahale@mozilla.com>
# Date: 2024-05-09 16:15:25
# Regressor Bug: 1824502
# File Overlap Count: 1
# Description:
#   Bug 1824502 - Implement native SVG filter rendering in WebRender r=gw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D209646
# ==============================================================================

diff -r 3e2ec3a3b738 -r c0f58fd74e56 gfx/wr/webrender_build/src/shader_features.rs
--- a/gfx/wr/webrender_build/src/shader_features.rs	Thu May 09 02:12:39 2024 +0000
+++ b/gfx/wr/webrender_build/src/shader_features.rs	Thu May 09 04:09:00 2024 +0000
@@ -79,6 +79,7 @@
         "cs_border_segment",
         "cs_border_solid",
         "cs_svg_filter",
+        "cs_svg_filter_node",
     ] {
         shaders.insert(name, vec![String::new()]);
     }